extensions/adobe/experience/decisioning/scope-details.schema.json

```diff
@@ -33,7 +32,40 @@
         "xdm:activity": {
           "$ref": "https://ns.adobe.com/experience/decisioning/activity-detail",
           "title": "Activity Detail for the Scope",
-          "description": "The activity that was a requested or chosen for this decision scope. Some scopes will prescribe the exact activity or mechanism (how) that makes the decision, other ."
+          "description": "The activity that was a requested or chosen for this decision scope. Some scopes will prescribe the exact activity (why and how) that makes the decision, other scopes will use a dynamic strategy determine which activity is used. Those scopes will list executionDetails for steps that were performed during decision-making."
+        },
+        "xdm:interactionMeasurements": {
+          "meta:status": "experimental",
+          "type": "array",
+          "items": {
+            "$ref": "https://ns.adobe.com/experience/decisioning/interaction-measurement"
+          },
+          "title": "Interaction Measurements used in this Scope",
+          "description": "The interaction measurements determine how inteaction event properties are interpreted (mapped) into outcomes for the propositions. Outcomes could be measured as the number of successes or failures or by a quantitative metric. An outcome can have a nominal value (e.g. success/failure), ordinal (rating value gathered) or numerical value (e.g. a measurable quantity)."
+        },
+        "xdm:strategy": {
```

Comment:
Should this be` xdm:strategies` (since it's an array?)

Reply:
I renamed it to the plural form